skimage/measure/tests/test_fit.py

```diff
@@ -435,9 +435,9 @@ def test_ransac_dynamic_max_trials():
     assert_equal(_dynamic_max_trials(50, 100, 8, 0.99), 1177)
     assert_equal(_dynamic_max_trials(50, 100, 8, 1), 9210)
 
-    # e = 0%, min_samples = 5
-    assert_equal(_dynamic_max_trials(1, 100, 5, 0), 0)
-    assert_equal(_dynamic_max_trials(1, 100, 5, 1), 360436504051)
+    # e = 0%, min_samples = 10
+    assert_equal(_dynamic_max_trials(1, 100, 10, 0), 0)
+    assert_equal(_dynamic_max_trials(1, 100, 10, 1), 162326183972299328)
```

Comment:
On main, it returns `-np.inf`.

Reply:
Can confirm. But why modify the existing test and not add another one precisely for the error case including a comment?

Reply:
With my first initial implementation, I thought that `min_samples = 5` would be redundant because the result for `min_samples=5` changes. But with your suggested change, the result doesn't change so that it makes sense to keep it. Just added it back with my new commit.